src/components/victory-axis.jsx

```diff
@@ -99,6 +99,7 @@ class VictoryAxis extends React.Component {
     if (this.props.labelPadding) {
       return this.props.labelPadding;
     }
+    // TODO: magic numbers
```

Comment:
They're magic numbers that make the padding look right!  I kind of figured on 20% over so that's the 1.2, and then doubled the multiplier for the labelPadding since it needs padding on both sides.